package.json

```diff
@@ -1,61 +1,80 @@
 {
   "name": "tinypool",
+  "type": "module",
   "version": "0.8.4",
+  "packageManager": "pnpm@9.0.6",
   "description": "A minimal and tiny Node.js Worker Thread Pool implementation, a fork of piscina, but with fewer features",
-  "type": "module",
-  "main": "./dist/esm/index.js",
-  "module": "./dist/esm/index.js",
+  "license": "MIT",
+  "homepage": "https://github.com/tinylibs/tinypool#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tinylibs/tinypool.git"
+  },
+  "bugs": {
+    "url": "https://github.com/tinylibs/tinypool/issues"
+  },
+  "keywords": [
+    "fast",
+    "worker threads",
+    "thread pool"
+  ],
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
+    },
+    "./entry/utils": {
+      "types": "./dist/entry/utils.d.ts",
+      "default": "./dist/entry/utils.js"
+    }
```

Comment:
I think these could be removed here. The only public API we provide is the entrypoint `dist/index.js`.